spec/models/manageiq/providers/nsxt/network_manager/refresher_spec.rb

```diff
@@ -2,4 +2,30 @@
   it ".ems_type" do
     expect(described_class.ems_type).to eq(:nsxt)
   end
+
+  context "#refresh" do
+    let(:ems) do
+      hostname = Rails.application.secrets.nsxt.try(:[], :hostname) || "NSXTHOSTNAME"
```

Comment:
should this be "NSXT_HOSTNAME" to be consistent with the rest?

Reply:
`       Validation failed: ManageIQ::Providers::Nsxt::NetworkManager: Hostname format is invalid.`